.github/workflows/dev-image-update.yaml

```diff
@@ -76,51 +76,30 @@ jobs:
     needs:
```

Comment:
I think you can remove the `env` section at the beginning of the file since you're explicit about defining it below.

Reply:
Future opportunity: remove file and directory exclusions